tool/teleport/common/teleport.go

```diff
@@ -231,6 +231,7 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	appStartCmd.Flag("insecure", "Insecure mode disables certificate validation").BoolVar(&ccf.InsecureMode)
 	appStartCmd.Flag("skip-version-check", "Skip version checking between server and client.").Default("false").BoolVar(&ccf.SkipVersionCheck)
 	appStartCmd.Flag("no-debug-service", "Disables debug service.").BoolVar(&ccf.DisableDebugService)
+	appStartCmd.Flag("mcp-demo-server", "Enables the Teleport Demo MCP server.").BoolVar(&ccf.MCPDemoServer)
```

Comment:
Can we expand the description to give a very brief overview of what it actually provides? "Enables the Teleport demo MCP server that can return current user information, cluster information and ..."